cqlsh_tests/test_cqlsh.py

```diff
@@ -2587,19 +2575,37 @@ def test_cjk_output(self):
         assert stdout_lines_sorted.find(expected) >= 0
 
 
+@reuse_cluster
 class TestCqlLogin(Tester, CqlshMixin):
     """
     Tests login which requires password authenticator
     """
+    def clear_auth(self, session):
+        # Clear auth after test to support node reusage
+        rows = list(session.execute(query="list roles", timeout=120))
+        for row in rows:
+            role = str(row.role)
+            if role != 'cassandra':
+                logging.debug("drop role if exists '" + role + "'")
+                session.execute(query="drop role if exists '" + role + "'", timeout=120)
+            session.cluster.control_connection.wait_for_schema_agreement(wait_time=120)
```

Comment:
Why are we waiting on a schema agreement here when we didn't change schema and just dropped specific roles?